src/oldBrowserSupportCheck.js

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2020 Splunk Inc.
```

Comment:
If you rebase to master, remove the header and regenerate it, you should start getting the current year